### DISCUSSION TITLE
Formatting

### DISCUSSION THREAD

**Discussion by larsnaesbye** [nav-solutions/rinex#63] [General]
Should we decide to keep a particular formatting regime, like `rustfmt`?

**Comment by gwbres** [comment]
Hello Lars,

> Should we decide to keep a particular formatting regime, like `rustfmt`?

yes we definitely should, and that is also related to this topic: https://github.com/gwbres/rinex/issues/59

I plan on running a `cargo fmt` over this crate in the next few weeks, probably when issue #59  is merged.   
`cargo fmt` is a great way to "respect" "standards", I have gotten used to use it when contributing to other projects

**Reply by larsnaesbye** [reply]
Good idea. `cargo fmt` and `rustfmt` are both fine with me.

**Reply by larsnaesbye** [reply]
https://github.com/gwbres/rinex/pull/64 is an attempt at a config file.

I don't personally see many other important things to change from the defaults, but [there are many parameters to change](https://rust-lang.github.io/rustfmt/) if we want to.

**Comment by larsnaesbye** [comment]
Closing this as I think we took a decision, and it's non-controversial.
